src/prototypes/article-listing/example/example.scss

```diff
@@ -0,0 +1,110 @@
+@use "../../../compiled/tokens/scss/color";
+@use "../../../compiled/tokens/scss/breakpoint";
+@use "../../../compiled/tokens/scss/size";
+@use '../../../mixins/fluid';
+@use '../../../mixins/media-query';
+@use "../../../mixins/ms";
+
+#article-listing {
+  .o-container {
+    padding-bottom: ms.step(6);
+    padding-top: ms.step(6);
+  }
+
+  .o-container:nth-child(even) {
+    background-color: #{color.$base-gray-lighter};
+  }
+
+  .title-content {
+    display: flex;
+    flex-direction: column;
+    grid-column: 1 / -1;
+    justify-content: space-between;
+  }
+
+  @media (min-width: breakpoint.$l) {
+    .title-content {
+      grid-column: auto;
+    }
+  }
+
+  .section-title {
+    font-weight: 300;
+    font-size: ms.step(5);
```

Comment:
Did you consider using one of our `fluid.*` mixins to set a `font-size` that changes between a min and max depending on viewport? These currently feel a little too large on small screens and this might remedy that. (See our heading component CSS for an example.)